app/controllers/books_controller.rb

```diff
@@ -0,0 +1,55 @@
+class BooksController < ApplicationController
+  def index
+    @books = Book.all
+  end
+
+  def show
+    @book = Book.find(params[:id])
```

Comment:
Since you use this line several times, you can DRY up your code by extracting it into a separate private method and calling that method instead. You can then also set it up as a before action scoped to show, edit, and update. Eg. 
```private

def find_book
  return Book.find(params[:id])
end```